lab1/Syntax.in

```diff
@@ -0,0 +1,37 @@
+Syntactic rules:
+program = "&START&" "&END&"
```

Comment:
shouldn't there be something in the program like at least a compound statement like
program = "&START" compound_statement "&STOP&" else your only allowed program is Start [Nothing] |Stop]